### ISSUE TITLE
Responsive

### ISSUE THREAD

**Issue by Valerie-Wissenburg** [slaterjen/web-iv-portfolio-site#3]
Looks sophisticated! ;) Unfortunately there were some things I noticed when I shrunk it down to medium screen. Your "project name" gets cut off. I don't even see half the word, it just disappears which maybe something you've already seen but just something to take note of.
Once you get these few fixed, you will be PROFICIENT/ADVANCED on the rubric.

**Comment by e-m-durham** [comment]
Yes, there are times where text is overlapping or not working correctly on certain screen sizes. But also the project page seems to have some issues where the content isn't being contained in a row so the line lengths are getting really long, etc. 

Considering these, I would assess responsiveness as unsatisfactory currently because the landing page issues could be potentially detrimental to the viewing experience on that screen size.

<img width="1401" alt="screen shot 2015-11-13 at 7 14 51 pm" src="https://cloud.githubusercontent.com/assets/2342978/11161153/ec3496f6-8a3a-11e5-9699-deccda79469d.png">
<img width="740" alt="screen shot 2015-11-13 at 7 15 14 pm" src="https://cloud.githubusercontent.com/assets/2342978/11161154/ec381f1a-8a3a-11e5-9d6b-aa3b034022fa.png">
